salt/journal/config/etc-nginx-traits.d-strict-transport-security.conf

```diff
@@ -0,0 +1,9 @@
+if ($https) {
+    add_header Strict-Transport-Security "max-age={{ 60 * 60 * 24 * 365 }}; includeSubDomains";
```

Comment:
http://demo--profiles.elifesciences.org/ (and every backend we expose temporarily for tests would need the SSL certificate) but unknown unknowns apply